### PR TITLE
fix: children param

### PR DESCRIPTION
**Pull request by grsabreu**
- [ ] feature
- [x] fix
- [ ] blocking
- [ ] docs

### CODE REVIEW

File: src/graphql-types/ReasonApolloSubscription.re [hunk 1]
```diff
@@ -41,39 +41,39 @@ module Get = (Config: ReasonApolloTypes.Config) => {
...
+    };
 
-  let make = (
-    ~variables: option(Js.Json.t)=?,
-    ~children: renderPropObj => ReasonReact.reactElement,
-    ) => 
+  let make =
+      (
+        ~variables: option(Js.Json.t)=?,
+        children: renderPropObj => ReasonReact.reactElement,
```
Comment (grsabreu) [inline src/graphql-types/ReasonApolloSubscription.re]: the fix

File: src/graphql-types/ReasonApolloSubscription.re [hunk 2]
```diff
@@ -41,39 +41,39 @@ module Get = (Config: ReasonApolloTypes.Config) => {
...
+          switch (Config.parse(data)) {
+          | parsedData => Some(parsedData)
+          | exception _ => None
+          }
+        },
+      error:
+        switch (apolloData##error |> Js.Nullable.toOption) {
+        | Some(error) => Some(error)
+        | None => None
+        },
```
Comment (grsabreu) [inline src/graphql-types/ReasonApolloSubscription.re]: this is refmt job

File: src/graphql-types/ReasonApolloSubscription.re (hunk 1 above)
Comment (Gregoirevda) [reply src/graphql-types/ReasonApolloSubscription.re]: oh waw! Good catch!

File: src/graphql-types/ReasonApolloSubscription.re [hunk 3]
```diff
@@ -1,28 +1,28 @@
 open ReasonApolloTypes;
 
-module Get = (Config: ReasonApolloTypes.Config) => {
-  [@bs.module] external gql : ReasonApolloTypes.gql = "graphql-tag";
-  [@bs.module "react-apollo"] external subscriptionComponent: ReasonReact.reactClass = "Subscription";
+module Make = (Config: ReasonApolloTypes.Config) => {
```
Comment (Gregoirevda) [inline src/graphql-types/ReasonApolloSubscription.re]: Breaking change: memo to myself to change all Get in Make

**Comment by Gregoirevda** [comment]
Thanks!! Looks perfect

**Pull request merged**